priv/proto/trace.proto

```diff
@@ -0,0 +1,329 @@
+// Copyright 2017, OpenCensus Authors
```

Comment:
Is suspect this is a copy fo an upstream file, it would be nice to note the exact source path and version of it.

Reply:
Good point. I prefer to commit the generated code just to make building easier (otherwise needs to always fetch the rebar3 gpb plugin and setup provider hooks...).

But with them committed maybe it is just the generated files that we need to reference trace.proto to and provide instructions in the readme on how to update the generated modules by fetching https://github.com/census-instrumentation/opencensus-proto/ to `priv/proto` and running `rebar3 protobuf`?

Reply:
that sounds good to me :)